### PR TITLE
Simplify Docker environment set-up for the local deployment of betting-house app

### PR DESCRIPTION
**Pull request by vasigorc**
Git commit message:

```shell
commit 1422c15669ba64e6d09426fbadcda47f5d6f8fa5
Author: Vasile Gorcinschi <vasilegorcinschi@Vasiles-MacBook-Pro.local>
Date:   Fri Sep 22 16:18:31 2023 -0400

    Simplified docker env set-up for local betting-house
    
    In this commit we've removed an extra-step of running SQL
    migration files once Docker postgres container is up. It
    has the advantage of abstracting away this tangental concern
    for people that ramp-up on Akka rather than on Docker.
```

### CODE REVIEW

File: betting-house/common-deployment/init_psql.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
```
Comment (franciscolopezsancho) [inline betting-house/common-deployment/init_psql.sh:3]:
```suggestion

apt-get update && \
apt-get install -y gosu && \
rm -rf /var/lib/apt/lists/*

```

File: betting-house/local-deployment/docker-compose.yml [hunk 2]
```diff
@@ -2,11 +2,14 @@ version: '2.2'
...
   postgres-betting-db:
     image: postgres:latest
+    volumes:
+      - ../common-deployment:/init-scripts
     ports:
       - 5432:5432
     environment:
       POSTGRES_USER: betting
       POSTGRES_PASSWORD: betting
+    entrypoint: /bin/bash -c "apt-get update && apt-get install -y gosu && /init-scripts/init_psql.sh"
```
Comment (franciscolopezsancho) [inline betting-house/local-deployment/docker-compose.yml]:
```suggestion
    entrypoint: /init-scripts/init_psql.sh
```

File: betting-house/README.md [hunk 3]
```diff
@@ -9,17 +9,10 @@ With the knowledge you have acquired so far, you should be able to improve the a
 
     cd local-deployment
 
-### start DB
+### start docker services
```
Comment (franciscolopezsancho) [inline betting-house/README.md:12]: Good point.

File: betting-house/local-deployment/docker-compose.yml (hunk 2 above)
Comment (franciscolopezsancho) [reply betting-house/local-deployment/docker-compose.yml:12]: Thank you very much for this improvement, Vasile. That's very kind of you.
I left a couple of suggestions you could follow. Just nitpicking. I hope you don't mind.

**Review by franciscolopezsancho — APPROVED** [review]
Thank you very much for this improvement, Vasile. That's very kind of you.
I left a couple of suggestions you could follow. Just nitpicking. I hope you don't mind.

**Comment by franciscolopezsancho** [comment]
let me know if you are fine with the changes.

**Comment by vasigorc** [comment]
> Thank you very much for this improvement, Vasile. That's very kind of you. I left a couple of suggestions you could follow. Just nitpicking. I hope you don't mind.

Thank you for the suggested improvements, I've merged them. JIC I've double checked that `docker compose up` starts with no issues and that the init scripts are executed. All is well ✅ 👍🏽 

```shell
...
postgres-betting-db-1  | 2024-06-24 22:13:59.593 UTC [1] LOG:  starting PostgreSQL 16.3 (Debian 16.3-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres-betting-db-1  | 2024-06-24 22:13:59.593 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres-betting-db-1  | 2024-06-24 22:13:59.593 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres-betting-db-1  | 2024-06-24 22:13:59.606 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-betting-db-1  | 2024-06-24 22:13:59.623 UTC [227] LOG:  database system was shut down at 2024-06-24 22:13:59 UTC
postgres-betting-db-1  | 2024-06-24 22:13:59.634 UTC [1] LOG:  database system is ready to accept connections
...
```

**Comment by franciscolopezsancho** [comment]
Thank you very much for your contribution @vasigorc :)

**Pull request merged**